client/src/reducers/index.js

```diff
@@ -1,11 +1,10 @@
 import { combineReducers } from 'redux';
 import { locationReducer } from './LocationReducer'
-import { ratingReducer } from './ratingReducer'
-
+import { ratingReducer } from './RatingReducer'
```

Comment:
@farrah1 - I believe this should still reference ratingReducer. At least on my side, the file is still named lowercased, and this upperCased reference breaks the app.

Reply:
@talyh I've changed the git config core.ignorecase to false as per your advice. This error should now be resolved.